spec/views/idv/inherited_proofing/retrieval.html.erb_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe 'idv/inherited_proofing/retrieval.html.erb' do
+  let(:flow_session) { {} }
+  let(:sp_name) { nil }
+  let(:locale) { nil }
+
+  before do
+    allow(view).to receive(:decorated_session).and_return(@decorated_session)
+    allow(view).to receive(:flow_session).and_return(flow_session)
+    allow(view).to receive(:url_for).and_return('https://www.example.com/')
+  end
+
```

Comment:
None of this setup appears to be necessary or relevant for the specs as currently implemented

```suggestion
```

Reply:
I was thinking forward, but I can remove them for now.